qa-tests-backend/src/main/groovy/util/Helpers.groovy

```diff
@@ -77,11 +81,6 @@ class Helpers {
         return "https://" + Env.mustGetHostname() + ":" + Env.mustGetPort()
     }
 
-    // withDo is like with, but returns a void so can safely be used in tests.
-    static void withDo(Object self, Closure closure) {
```

Comment:
Removed as I could not sort it out with types. Probably we need to pass template param but we can remove it and just use `with` as `safely` in comment means it does not return a value which is not a problem unless you return `false`